test/e2e/apimachinery/health_handlers.go

```diff
@@ -42,7 +42,7 @@ var (
 		"[+]poststarthook/start-apiextensions-informers ok",
 		"[+]poststarthook/start-apiextensions-controllers ok",
 		"[+]poststarthook/crd-informer-synced ok",
-		"[+]poststarthook/bootstrap-controller ok",
```

Comment:
paranoid question, do you think somebody may be depending on this name?

Reply:
We never gave any guarantees about these and changed them at will.

Reply:
But no strong opinion. We can also keep the old name. Wdyt?

Reply:
@wojtek-t or @liggitt WDYT?

Reply:
I hope noone relies on it, but given it doesn't matter much for us, I would actually leave the name as it was previously.